### ISSUE TITLE
Report

### ISSUE THREAD

**Issue by AlexandruDoda** [avocode/avocode#1210]
IMPORTANT: Please try to describe how did it happen.

-----
App v. `2.10.1`
Source UID: 651
```
Error: Object has been destroyed
Error: Object has been destroyed
    at Error (native)
    at EventEmitter.<anonymous> (/Applications/Avocode.app/Contents/Resources/electron.asar/browser/rpc-server.js:325:54)
    at emitThree (events.js:116:13)
    at EventEmitter.emit (events.js:194:7)
    at WebContents.<anonymous> (/Applications/Avocode.app/Contents/Resources/electron.asar/browser/api/web-contents.js:218:13)
    at emitTwo (events.js:106:13)
    at WebContents.emit (events.js:191:7)
    at metaToValue (/Applications/Avocode.app/Contents/Resources/electron.asar/renderer/api/remote.js:176:13)
    at descriptor.get [as webContents] (/Applications/Avocode.app/Contents/Resources/electron.asar/renderer/api/remote.js:125:16)
    at t.terminate (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:24:4193)
    at e._terminateTaskProcess (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:24:9583)
    at e.dispose (file:///Applicatio
```

**Comment by connor-bean** [comment]
Hey @AlexandruDoda -

It looks like this is probably an Electron error, which we don't have the ability to troubleshoot. Could you try updating to our 2.12.0 beta to see if it helps? You're currently running 2.10.1.

You can download 2.12.0 here:

mac: https://upx.cz/B16
win x64: https://upx.cz/B1L
linux x64: https://upx.cz/B1n

Please let us know what happens once you update to 2.12.0.

Thanks,
Connor, Avocode Support

**Comment by connor-bean** [comment]
Please let us know if this was fixed in 2.12.1.

Thanks,
Connor, Avocode Support
